wpilibcExamples/src/main/cpp/examples/ElevatorSimulation/cpp/Robot.cpp

```diff
@@ -53,7 +72,15 @@ class Robot : public frc::TimedRobot {
   frc::DCMotor m_elevatorGearbox = frc::DCMotor::Vex775Pro(4);
 
   // Standard classes for controlling our elevator
-  frc2::PIDController m_controller{kElevatorKp, 0, 0};
+  frc::TrapezoidProfile<units::meters>::Constraints m_constraints{2.45_mps,
+                                                                  2.45_mps_sq};
+  frc::ProfiledPIDController<units::meters> m_controller{
+      kElevatorKp, kElevatorKi, kElevatorKd, m_constraints};
+
+  frc::ElevatorFeedforward feedforward(kElevatorkS,
+                                       kElevatorkG,
+                                       kElevatorkV,
+                                       kElevatorkA);
```

Comment:
You have to use curly braces here to write constructor arguments in the header.
```suggestion
  frc::ElevatorFeedforward feedforward{kElevatorkS,
                                       kElevatorkG,
                                       kElevatorkV,
                                       kElevatorkA};
```